patches/chromium/fix_desktop_capturer_show_own_window.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: George Xu <george.xu@slack-corp.com>
+Date: Wed, 11 Dec 2024 15:00:09 -0800
+Subject: fix: desktopCapturer.getSources does not return electron window on
+ Windows
+
+This patch is to fix issue https://github.com/electron/electron/issues/36037, where `desktopCapturer.getSources` does not return the electron window on Windows.
+
+This patch can be removed once we deprecate the current implementation of desktopCapturer.
```

Comment:
Is there a tracking issue for this ?

Reply:
There's no Chromium issue afaik

Reply:
@deepak1556 We have a project board for it, but no singular issue, I'll create one - the TL;DR is, there's no plan on deprecating the old API anytime in the near future, but we do want to create a new API that uses the getDisplayMedia web API 🙂

Reply:
@georgexu99 I think technically though, this _could_ happen independently of whether or not we're calling `getSources` only, is that right? Maybe we could clarify this to say something like:
```suggestion
This patch likely cannot be upstreams. It can be removed if we can expose the ShouldEnumerateCurrentProcessWindows method in Electron's desktopCapturer API, or if desktopCapturer can use an alternative source API for getSources on Windows.
```

Reply:
Good revision!